.github/workflows/release-packages.yml

```diff
@@ -0,0 +1,68 @@
+name: Release Packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      packages:
+        description: 'Packages to release (comma-separated)'
+        required: true
+        type: string
+        default: '@novu/js,@novu/react,@novu/nextjs,@novu/react-native'
```

Comment:
How does dependencies work? For example if next/js is dependant on js and react and js version? Will all be bumped in the same time properly?